From: Chris Leech <cleech@redhat.com>
Subject: qedi.c: Removed unused linux/ethtool.h

Bugzilla: ZZZ
Upstream Status:
Build Info: XXX
Tested:

commit 197713ad7e3e944102bbd792e1ab9ec4a67100c0
Author: Khem Raj <raj.khem@gmail.com>
Date:   Fri Feb 2 23:25:21 2018 -0800

    qedi.c: Removed unused linux/ethtool.h
    
    Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 iscsiuio/src/unix/libs/qedi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/iscsiuio/src/unix/libs/qedi.c b/iscsiuio/src/unix/libs/qedi.c
index b81fecdfe78f..24cb89ad3580 100644
--- a/iscsiuio/src/unix/libs/qedi.c
+++ b/iscsiuio/src/unix/libs/qedi.c
@@ -49,7 +49,6 @@
 #include <arpa/inet.h>
 #include <linux/types.h>
 #include <linux/sockios.h>
-#include <linux/ethtool.h>
 #include <linux/netlink.h>
 #include <sys/mman.h>
 #include <sys/ioctl.h>
